Core/Core/Quizzes/QuizDetails/Teacher/ViewModel/TeacherQuizDateSectionViewModelLive.swift

```diff
@@ -18,7 +18,7 @@
 
 import SwiftUI
 
-public class QuizDateSectionViewModel: DateSectionViewModelProtocol {
+public class TeacherQuizDateSectionViewModelLive: DateSectionViewModelProtocol {
```

Comment:
**Not blocking.** 
In Canvas code, `Live` suffix is mostly used for Interactors. I'd prefer selecting another suffix, perhaps omitting it altogether to avoid confusion.